less/index.less

```diff
@@ -54,6 +54,110 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
```

Comment:
Same as above, many devs believe having an empty line between each selector makes the code easier to read.